crml/governance/src/mock.rs

```diff
@@ -0,0 +1,163 @@
+* You may obtain a copy of the License at the root of this project source code,
+* or at:
+*     https://centrality.ai/licenses/gplv3.txt
+*     https://centrality.ai/licenses/lgplv3.txt
+*/
+
+use crate as crml_governance;
+use cennznet_primitives::types::{AssetId, Balance};
+use crml_generic_asset::StakingAssetCurrency;
+use crml_support::StakingAmount;
+use frame_support::{parameter_types, traits::RegistrationInfo, weights::Weight, PalletId};
+use frame_system::EnsureRoot;
+use sp_core::H256;
+use sp_runtime::{
+	testing::Header,
+	traits::{BlakeTwo256, IdentityLookup},
+	Perbill,
+};
+
+pub type AccountId = u64;
+
+type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Test>;
+type Block = frame_system::mocking::MockBlock<Test>;
+
+frame_support::construct_runtime!(
+	pub enum Test where
+		Block = Block,
+		NodeBlock = Block,
+		UncheckedExtrinsic = UncheckedExtrinsic,
+	{
+		System: frame_system::{Module, Call, Config, Storage, Event<T>},
+		Scheduler: pallet_scheduler::{Module, Call, Config, Storage, Event<T>},
+		GenericAsset: crml_generic_asset::{Module, Call, Storage, Config<T>, Event<T>},
+		Governance: crml_governance::{Module, Call, Storage, Event},
+	}
+);
+
+parameter_types! {
+	pub const BlockHashCount: u64 = 250;
+	pub BlockWeights: frame_system::limits::BlockWeights =
+		frame_system::limits::BlockWeights::simple_max(1_000_000);
+}
+impl frame_system::Config for Test {
+	type BlockWeights = ();
+	type BlockLength = ();
+	type BaseCallFilter = ();
+	type Origin = Origin;
+	type Index = u64;
+	type BlockNumber = u64;
+	type Call = Call;
+	type Hash = H256;
+	type Hashing = BlakeTwo256;
+	type AccountId = AccountId;
+	type Lookup = IdentityLookup<Self::AccountId>;
+	type Header = Header;
+	type BlockHashCount = BlockHashCount;
+	type Event = Event;
+	type DbWeight = ();
+	type Version = ();
+	type PalletInfo = PalletInfo;
+	type AccountData = ();
+	type OnNewAccount = ();
+	type OnKilledAccount = ();
+	type SystemWeightInfo = ();
+	type SS58Prefix = ();
+}
+
+parameter_types! {
+	pub const TreasuryPalletId: PalletId = PalletId(*b"py/trsry");
+}
+impl crml_generic_asset::Config for Test {
+	type AssetId = AssetId;
+	type Balance = Balance;
+	type Event = Event;
+	type OnDustImbalance = ();
+	type WeightInfo = ();
+}
+
+parameter_types! {
+	pub const MaxScheduledPerBlock: u32 = 50;
+	pub MaximumSchedulerWeight: Weight = Perbill::from_percent(80) * 100;
+
+}
+impl pallet_scheduler::Config for Test {
+	type Event = Event;
+	type Origin = Origin;
+	type PalletsOrigin = OriginCaller;
+	type Call = Call;
+	type MaximumWeight = MaximumSchedulerWeight;
+	type ScheduleOrigin = EnsureRoot<AccountId>;
+	type MaxScheduledPerBlock = ();
+	type WeightInfo = ();
+}
+
+pub struct MockStakingAmount;
+impl StakingAmount for MockStakingAmount {
+	type AccountId = AccountId;
+	type Balance = Balance;
+
+	fn active_balance(controller: Self::AccountId) -> Self::Balance {
+		match controller {
+			1 => 10000,
+			_ => 20000,
```

Comment:
would make missing accounts 0 so you don't get confusing test results
also needs the extra 4 `0`s
```suggestion
			1 => 100000000,
			2 => 200000000,
			_ => 0,
```